### DISCUSSION TITLE
3.0版本定时配置

### DISCUSSION THREAD

**Discussion by zxcseraph** [apache/dolphinscheduler#9958] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/36960043/167442216-3d8abfbc-d3ec-4dc3-8a6a-891128a7704a.png)
如图，在配置定时的时候，查看《接下来五次执行时间》，发现比上面配
![aaaa](https://user-images.githubusercontent.com/36960043/167442593-8eb0e0c3-1406-49b8-9caf-397b78c4383e.jpg)
置的时间早4个小时。

**Comment by songjianet** [comment, marked as answer]
It may be a time zone issue. Please provide the time zone selected by the current user. @zxcseraph
